go-controller/pkg/libovsdbops/model_client.go

```diff
@@ -113,6 +113,9 @@ type OperationModel struct {
 	// DoAfter is invoked at the end of the operation and allows to setup a
 	// subsequent operation with values obtained from this one.
 	DoAfter func()
+	// Name is used to signify if this model has a name being used. Typically
+	// corresponds to Name field used on ovsdb objects
+	Name string
```

Comment:
Can you add a bit more wording why this is special?

Reply:
sure